shared/pingpong/pingpong.go

```diff
@@ -882,7 +883,7 @@ func (pps *WorkerState) sendFromTo(
 			timeCredit -= took
 			if timeCredit > 0 {
 				time.Sleep(timeCredit)
```

Comment:
Why not use `NanoSleep()` here?

Reply:
No need to be super-accurate here, since the  timeCredit mechanism would compensate by dropping sleeps on subsequent iterations.